teamC/dayoung/models/postDao.js

```diff
@@ -0,0 +1,106 @@
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch((err) => {
+    console.error("Error occurred during Data Source initialization", err);
+	  myDataSource.destroy();
+  });
+
+  /*********************************************************** */
+
+const createPost = async ( title, content, userId ) => {
+	try {
+		return await myDataSource.query(
+		`INSERT INTO posts(
+		    title,
+            content,
+            user_id
+		) VALUES (?, ?, ?);
+		`,
+		[ title, content, userId ]
+
+	  );
+	} catch (err) {
+		const error = new Error('INVALID_DATA_INPUT');
+		error.statusCode = 500;
+		throw error;
+	}
+};
+
+const DataPost = async ( postId) => {
```

Comment:
전체적으로 함수 이름들 한 번씩 확인해 주세요.
```suggestion
const getPostList = async ( postId) => {
```

Reply:
수정했습니다.